db/get_all_recipes_by_kitchen_id.sql

```diff
@@ -0,0 +1,4 @@
+SELECT r.*
+FROM kitchen_recipes kr 
+JOIN recipes r on r.id = kr.recipe_id
+WHERE kr.kitchen_id = ${id};
```

Comment:
A more common way to write this is to have the `FROM` be the table that you selecting the bulk of the columns from 
```suggestion
SELECT r.*
FROM recipes r 
JOIN kitchen_recipes kr on kr.recipe_id = r.id
WHERE kr.kitchen_id = ${id};
```

Reply:
Resolved!